go.mod

```diff
@@ -2,8 +2,10 @@ module go.temporal.io/server
 
 go 1.21
 
+replace go.temporal.io/api => ../api-go
```

Comment:
This will be removed once https://github.com/temporalio/api-go/pull/146 is merged